### ISSUE TITLE
[Help] "NewRandomizerGUI has been compiled by a more recent version of Java Runtime"

### ISSUE THREAD

**Issue by Rukir** [Ajarmar/universal-pokemon-randomizer-zx#644]
I have the latest version of Java Runtime downloaded as provided by the site (1.8.0_371), on 64 bit, and I have tried un- and re-installing it a few times to no avail. This prevents me from accessing the randomizer. Any idea what's going on?

![image_2023-07-07_165435358](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/assets/87564035/96daf589-55df-4dc4-98bf-b73aac05d8dd)

![image](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/assets/87564035/293672a9-6591-4f0d-8d87-2dc6849127a3)

**Comment by tom-overton** [comment]
Are you using the version of the randomizer available on the [releases](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/releases/tag/v4.6.0) page? When I inspect it, I see it should work:
```
Classfile /G:/NewRandomizerGUI.class
  Last modified Feb 10, 2023; size 192898 bytes
  MD5 checksum a129010b68263e1dd5116c62819e2d77
  Compiled from "NewRandomizerGUI.java"
public class com.dabomstew.pkrandom.newgui.NewRandomizerGUI
  minor version: 0
  major version: 52
```
At the very least, it definitely was *not* compiled with version 62.0

**Comment by tom-overton** [comment]
If you're using this fork: https://github.com/voliol/universal-pokemon-randomizer
You'll need to figure out how to get help from them, not us. I downloaded it, and this version indeed was compiled with version 62.0. Another user also ran into an issue with this fork and said that they don't have an issue tracker, so unfortunately I'm not sure how you're supposed to get help, but that being said, we can't really help you with problems that originate from forks of our code. We only know how to support the code that we wrote.
